src/ui/components/ShowMoreCard/index.js

```diff
@@ -1,55 +1,125 @@
+/* @flow */
 /* eslint-disable react/no-danger */
-
+import invariant from 'invariant';
 import makeClassName from 'classnames';
 import * as React from 'react';
-import PropTypes from 'prop-types';
 import { compose } from 'redux';
 
 import translate from 'core/i18n/translate';
+import withUIState from 'core/withUIState';
 import { sanitizeHTML } from 'core/utils';
 import Card from 'ui/components/Card';
+import type { I18nType } from 'core/types/i18n';
 
 import './styles.scss';
 
 // This refers to height of `Card-contents` CSS class,
 // beyond which it will add read more link.
 export const MAX_HEIGHT = 150;
 
-export class ShowMoreCardBase extends React.Component {
-  static propTypes = {
-    children: PropTypes.node,
-    className: PropTypes.string,
-    header: PropTypes.node,
-    i18n: PropTypes.object.isRequired,
-  };
+type UIStateType = {|
+  showAllContent: boolean,
+  readMoreExpanded: boolean,
+|};
+
+type Props = {|
+  children: any,
```

Comment:
Why does this need to be `any`? Could it be 'React.Node`?

Reply:
@bobsilverberg I used this to fix the issue mentioned above:
https://github.com/mozilla/addons-frontend/pull/5874#discussion_r217543489

I seems I can also do the following if it's better:
`children: React.Element<any> | Object,`

Reply:
I just tried your code with just `React.Element<any>` (i.e., not `React.Element<any> | Object`) and I didn't get any Flow errors. Does that not work for you? `Object` isn't really much better than `any`.